homeassistant/components/zwave_js/light.py

```diff
@@ -418,10 +418,7 @@ def _calculate_color_support(self) -> None:
         if ww_val and cw_val:
             self._supports_color_temp = True
         # only one white channel (warm white) = rgbw support
-        elif red_val and green_val and blue_val and ww_val:
-            self._supports_rgbw = True
-        # only one white channel (cool white) = rgbw support
```

Comment:
We should probably preserve this comment here, too.

Reply:
Sharp, didn't spot the warm/cool